src/components/organisms/News.tsx

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
 import { AnywayLink, Text, TextType } from '../atoms';
-import { Box } from '@material-ui/core';
+import { Box, makeStyles } from '@material-ui/core';
 import { borderColor } from '../../style/_globals';
 import { useStore } from '../../store/storeConfig';
 import RootStore from '../../store/root.store';
 import { observer } from 'mobx-react-lite';
 
+const useStyles = makeStyles({
+  container:{
+    maxHeight: "300px"
+  }
+});
+
 const News: React.FC = () => {
   const store: RootStore = useStore();
+  const classes = useStyles();
 
   return (
-    <Box flexDirection={'column'}>
+    <Box className={classes.container} flexDirection={'column'}>
       {store.newsFlashCollection.length > 0 ?
       store.newsFlashCollection.map((news) => {
-        const date: null | string = news.date == null ? '' : new Date(news.date).toLocaleDateString();
+        const date: null | string = news.date == null ? '' : new Date(news.date.replace(/-/g, "/")).toLocaleDateString();
```

Comment:
we use single quotes on code. although inside a template, when between curtly crackets - style should be like `javascript / typescript`. (only actual `JSX` tags uses double quotes).
replace `"/"` with `'/'`